app/models/rental.rb

```diff
@@ -0,0 +1,21 @@
+class Rental < ApplicationRecord
+  belongs_to :movie
+  belongs_to :customer
+
+  validates :customer_id, presence: true
+  validates :movie_id, presence: true
+
+  def checked_out?()
+    return self.checkedout == true
```

Comment:
because of the nature of this conditional, you could probably just return `self.checkedout`